src/data/hi.yml

```diff
@@ -479,32 +493,32 @@ community:
   sharing3: "के तहत लाइसेंस प्राप्त है। कृपया बेझिझक साझा करें और आरोपण के साथ रीमिक्स करें। "
 
   contribute-title: "योगदान"
-  contribute1: "हमारा समुदाय हमेशा सभी अलग-अलग तरीकों से मदद करने के लिए उत्साही लोगों की तलाश में रहता है।"
+  contribute1: "हमारा समुदाय हमेशा उत्साही लोगों की तलाश में रहता है जो हर तरह से मदद करें।"
   develop-title: "विकास करना।"
   develop1: "GitHub"
-  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों को प्रलेखित किया जाता है, और कोड के बारे में चर्चा होती है। आरंभ करने के लिए "
+  develop2: " मुख्य स्थान है जहां कोड एकत्र किया जाता है, मुद्दों का दस्तावेजीकरण किया जाता है, और कोड के बारे में चर्चा की जाती है। आरंभ करने के लिए "
   develop3: " विकास शिक्षण"
   develop4: " देखें, या "
   develop5: "अपना स्वयं का लाइब्रेरी बनाएं।"
   document-title: "दस्तावेज़।"
-  document1: " दस्तावेज़ीकरण का शौक सभी को होता है। "
+  document1: " सभी को दस्तावेज़ीकरण पसंद है। मदद की जरूरत है "
   document2: "उदाहरणों को पोर्ट करने"
   document3: ", और"
   document4: " प्रलेखन को जोड़ने"
-  document5: ", और शिक्षण बनाने में मदद की आवश्यकता है।"
+  document5: ", और शिक्षण बनाने में।"
   teach-title: "सिखाओ।"
-  teach1: " एक कार्यशाला, एक कक्षा, एक दोस्त, एक सहयोगी को सिखाओ! ट्विटर पर @ p5xjs टैग करें और आप जो कर रहे हैं उसे साझा करने के लिए हम पूरी कोशिश करेंगे।"
+  teach1: " एक कार्यशाला, एक कक्षा, एक दोस्त, एक सहयोगी को पढ़ाएं! ट्विटर पर @ p5xjs टैग करें और आप जो कर रहे हैं उसे साझा करने के लिए हम पूरी कोशिश करेंगे।"
   create-title: "सर्जन करना।"
-  create1: " p5.js डिजाइनरों, कलाकारों, कोडर्स, प्रोग्रामर की तलाश कर रहा है ताकि फ्रंट पेज पर आपके रचनात्मक और अद्भुत काम को दिखाया जा सके और अन्य लोगों को प्रेरित किया जा सके। अपना काम "
+  create1: " p5.js आपके रचनात्मक और अद्भुत काम को सामने वाले पृष्ठ पर दिखाने और अन्य लोगों को प्रेरित करने के लिए डिजाइनरों, कलाकारों, कोडर्स, प्रोग्रामर की तलाश में है। अपना काम "
   create2: "hello@p5js.org"
   create3: "पर भेजें।"
   donate-title: "दान देना।"
   donate1: " p5.js स्वतंत्र और खुला स्रोत है और कलाकारों द्वारा बनाया गया है। "
   donate2: "प्रोसेसिंग फाउंडेशन"
-  donate3: " को दान के माध्यम से p5.js के विकास में सहायता करें।"
-  contributors-conference-title: "योगदानकर्ता सम्मेलन"
-  contributors-conference1: "जबकि अधिकांश कार्य ऑनलाइन होते है, हम IRL भी आयोजित करते हैं। हमने पिट्सबर्ग, पीए में  "
-  contributors-conference2: " कार्नेगी मेलन विश्वविद्यालय  में योगदानकर्ताओं के दो सम्मेलन आयोजित किए। कलाकार, डिजाइनर, डेवलपर्स, शिक्षक p5.js परियोजना को आगे बढ़ाने के लिए एक साथ शामिल हुए। "
+  donate3: " को दान के माध्यम से p5.js के विकास में सहायता करें। "
+  contributors-conference-title: "p5.js योगदानकर्ता सम्मेलन"
+  contributors-conference1: "जबकि अधिकांश कार्य ऑनलाइन होते है, हम IRL भी आयोजित करते हैं। हमने पिट्सबर्ग, पीए में "
```

Comment:
For Pittsburgh, PA you can write full form of PA.
Actually it's  less common practice to abbreviate places name in hindi.

Reply:
ok I will make to Pittsburgh, Pennsylvania

Reply:
Better